console.go

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) 2014-2019 MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the license is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+ package gowin32
+
+ import (
+ 	 "github.com/winlabs/gowin32/wrappers"
+ )
+
+ func EnableVTSequences() error {
+	err := addStdConsoleModeFlag(
+		wrappers.STD_INPUT_HANDLE,
+		wrappers.ENABLE_VIRTUAL_TERMINAL_INPUT)
```

Comment:
This is new with Windows 10 so you will have to careful when you call it.

Also, verify Ctrl-C still works for your application

Reply:
I already removed `ENABLE_VIRTUAL_TERMINAL_INPUT` in a subsequent commit because it was causing problems.